django/cantusdb_project/templates/base.html

```diff
@@ -446,7 +446,7 @@ <h5>Login for Contributors</h5>
             <a href="mailto:debra.lacoste@dal.ca">
                 <img src="{% static "sendMailIcon.png" %}" alt="Send us email" loading="lazy">
             </a>
-            <small style="float: right">Cantus Database © 2012-2023</small>
+            <small style="float: right">Cantus Database © 2012-2024</small>
```

Comment:
```suggestion
            <small style="float: right">Cantus Database © 2012-{% now "Y" %}</small>
```

To save yourself or somebody else some trouble next year. :-D

https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#now

Reply:
Brilliant!